package.json

```diff
@@ -26,6 +26,8 @@
   },
   "dependencies": {
     "chalk": "^1.1.3",
-    "minimist": "^1.2.0"
+    "htmltojsx": "git://github.com/harrison/react-magic.git",
```

Comment:
Groovy, I will use the enhancement tag moving forward. And I will definitely link through the site once its ready!